package.json

```diff
@@ -41,6 +41,7 @@
     "cheerio": "^0.22.0",
     "fibers": "^4.0.2",
     "handlebars": "^4.5.3",
+    "isomorphic-fetch": "^2.2.1",
```

Comment:
`isomorphic-fetch` uses `node-fetch` on Node.js so why not using `node-fetch` directly?

Reply:
Hey, thanks! Thanks a great call and a good catch. I've updated the code and rebased it to the latest commit on `master`